recipes/aicspylibczi/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "aicspylibczi" %}
+{% set version = "3.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/aicspylibczi-{{ version }}.tar.gz
+    sha256: b8c2e9360e47ad9c08d9af492bea532daa93f6b73d596fadf683230b212c24e7
```

Comment:
I guess this needs to update to match the sha256 of the actual tar.gz.   My computer says `d903caaac5576922f8dd4b1170fae3421fb9d203128b4d8ecb11681173e7046c`